packages/medusa/src/api-v2/store/products/query-config.ts

```diff
@@ -0,0 +1,41 @@
+export const defaultStoreProductFields = [
+  "id",
+  "title",
+  "subtitle",
+  "description",
+  "handle",
+  "is_giftcard",
+  "discountable",
+  "thumbnail",
+  "collection_id",
+  "type_id",
+  "weight",
+  "length",
+  "height",
+  "width",
+  "hs_code",
+  "origin_country",
+  "mid_code",
+  "material",
+  "created_at",
+  "updated_at",
+  "*type",
+  "*collection",
+  "*options",
+  "*options.values",
+  "*tags",
+  "*images",
+  "*variants",
+  "*variants.options",
+]
+
+export const retrieveProductQueryConfig = {
+  defaults: defaultStoreProductFields,
```

Comment:
q: do we need to limit what is accessible by the store, I am thinking if someone add more link or properties or something, the store could request for some admin data? This is more of a global question :)